src/pages/Dashboard/Markets/CollateralConfirmModal.tsx

```diff
@@ -0,0 +1,35 @@
+/** @jsxImportSource @emotion/react */
+import React from 'react';
+import { Icon, Modal, IModalProps } from 'components';
+import { Asset } from 'types';
+import { ReactComponent as LogoWithText } from 'assets/img/v2/venusLogoWithText.svg';
+import Typography from '@mui/material/Typography';
+import { useStyles } from './styles';
+
+export interface IConfirmCollateralModalProps {
+  asset: Asset | undefined;
```

Comment:
Very small thing, feel free to ignore it:
```suggestion
  asset?: Asset;
```